pkg/scheduler/framework/arguments.go

```diff
@@ -99,3 +99,9 @@ func GetArgOfActionFromConf(configurations []conf.Configuration, actionName stri
 
 	return nil
 }
+
+// KeyExists checks if the key exists in the Arguments map
+func (a Arguments) KeyExists(key string) bool {
```

Comment:
it is redundant.

Reply:
ditto

Reply:
Dont need. Please refer to predicate.Proportional plugin.

Reply:
I think using this method makes the code easier to read for others. It also helps keep the code cleaner. And I think the `KeyExists` method is common and necessary in the configuration file. Other callers can also use this method.